docs/usermanual/index.md

```diff
@@ -17,16 +17,156 @@ TODO
 
-TODO
+Making copies of encrypted resources is a bit more involved. By default, encrypted Elastic Block Store (EBS) resources 
+use the aws/ebs key that is associated with the AWS account. Snapshots that are encrypted with this key cannot be shared 
+directly. Instead, you must first make a copy of the snapshot, and re-encrypt it with a Customer Managed Key (CMK) key. 
+Both the CMK key and the snapshot copy need to then be shared with the destination account.
+
+
+To reduce key management hassle, libcloudforensics allows you to transfer encrypted EBS resources between accounts by 
+generating a one-time use CMK key. This key is deleted once the process completes. The process is depicted below:
+
+![ebs-encryption](https://lh4.googleusercontent.com/U_DxaWaZqx00gSG8kfhlwB6dkGGcPi5_Cd_qgZR9VMdW0rWmrHD-VDeChMKNBfJJ86qBJC9duaZzHbmBxWU1WpsNkNYmp3UQvmU2R9EuDmn0vJJqy0rZVTjZVBePIXvb8QlJmBgq)
+
+All of the code snippets and command lines given in the previous section can be applied as-is, regardless of the target
+volume using EBS encryption or not.
 
 ### Analysis VMs
 
-TODO
+Use case: you have a volume `vol1` in an AWS account (Account A, in zone `us-east-2b`) which you would like to make a copy 
+of in a different account (Account B). Afterwards, you want to start an analysis VM in Account B, and attach the disk 
+copy that you created to start with your investigation.
+
+Your AWS credentials configuration should be similar to what is described above.
+
+#### Using the library
+
+```python
+from libcloudforensics.providers.aws import forensics
+
+# Create a forensic copy of the volume 'vol1' from one account to another 
+
+copy = forensics.CreateVolumeCopy('us-east-2b',
+                                  volume_id='vol1', 
+                                  src_profile='src_profile', 
+                                  dst_profile='dst_profile')
+
+# Start an analysis VM 'vm-forensics' for investigation in the AWS account 
+# dst_profile, and attach the copy created in the previous step.
+
+analysis_vm, _ = forensics.StartAnalysisVm('vm-forensics',
+                                           'us-east-2b', 
+                                            50, 
+                                            cpu_cores=4,
+                                            attach_volumes=[(copy.volume_id, '/dev/sdf')], 
+                                            dst_profile='dst_profile')
+```
+
+You're now ready to go! Log in your AWS account, find the instance (you can search it based on the name tag 
+`vm-forensics`) and click on `Connect`.
+
+``` important:: Pro tip: you can export an environment variable 'STARTUP_SCRIPT' that points to a custom bash script of
+    yours. This script will be shipped to the instance being created and executed during the first boot. You can 
+    therefore do any kind of pre-processing you want in this script.
+```
+
+#### Using the CLI 
+
+The volume copy CLI is identical to what is presented in previous sections.
```

Comment:
can we just add the cli cmd that copies the volume here before creating the analysis VM, this makes it consistent to the python example and clearer that the volume must be residing under the destination profile where you will start the VM.